appjs/views/edit_project.js

```diff
@@ -325,6 +393,19 @@ var EditProject = BaseView.extend(require('./mixins/actions'), require('./mixins
 
   afterRender: function() {
     var view = this, promises = [];
+    view.showPreviewButtons();
+    $(window).resize(function(){
+      view.showPreviewButtons();
+      if(view.formWidth){
+        if($(window).width() > 768){
+          $('#form-pane').css("width", view.formWidth);
+          $('#preview-pane').css("width", $(window).width() - view.formWidth);
+        } else {
+          $('#form-pane').css("width", '100%');
+          $('#preview-pane').css("width", '100%');
+        }
+      }
+    });
```

Comment:
Retain the width of the form panel that the user has set even if the window is resized.